webrender/src/scene_builder.rs

```diff
@@ -150,11 +152,14 @@ impl SceneBuilder {
                     build_scene(&self.config, request)
                 });
 
-                let rasterized_blobs = blob_rasterizer.as_mut().map_or(
+                let mut more_rasterized_blobs = blob_rasterizer.as_mut().map_or(
                     Vec::new(),
                     |rasterizer| rasterizer.rasterize(&blob_requests),
                 );
 
+                rasterized_blobs.append(&mut more_rasterized_blobs);
```

Comment:
this looks weird. Isn't `rasterized_blobs` just `Vec`? Can't the argument just be `more_rasterized_blobs` without `&mut`?